<ui version="4.0" >
 <class>UpdateOptionsDialogBase</class>
 <widget class="QDialog" name="UpdateOptionsDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update/Revert to Release/Branch/Date</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="headRadio" >
        <property name="text" >
         <string>&amp;Most recent from current branch</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="revisionRadio" >
          <property name="text" >
           <string>An arbitrary &amp;revision/tag/branch:</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="revisionEdit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Type your release name here (leave empty for HEAD)</string>
          </property>
          <property name="whatsThis" >
           <string>Fill the field with the release or branch name (e.g. &lt;i>make_it_cool, kdevelop_alpha5, ...&lt;/i>)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QRadioButton" name="dateRadio" >
          <property name="text" >
           <string>An arbitrary &amp;date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="dateEdit" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>FIll the field with a date (e.g. &lt;i>20030204&lt;/i>)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="revertCheck" >
        <property name="text" >
         <string>&amp;Enforce even if the file has been locally modified (revert)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateOptionsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateOptionsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revisionRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>revisionEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>138</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>83</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel" >
     <x>133</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
